src/_hidden.scss

```diff
@@ -0,0 +1,27 @@
+// Example @include hidden( disappear )
+@mixin hidden( $state: disappear ) {
+	@if $state == disappear {
+		display: none !important;
```

Comment:
Adding `!important` here will make hard to interact with elements that use this mixin in their rules to hide them. Could this be removed or added as an optional argument instead?

Found this snippet on the web which could be useful for this instance:
```SCSS
@function if-important($important){
  @return #{if($important, '!important', '')};
}

@mixin hidden( $state: disappear, $important: false ) {
    display: none if-important( $important );
    visibility: hidden;
}
```

What do you think? CC @gooklani

Reply:
Thank you @kuserich,
We need to check it (form block) on the WordPress default themes and our themes to decide about the important property 🧐

Reply:
We could nest this mixin a few levels deeper when applying to increase specificity and remove the "important" keyword as a result. What do you guys think? @kuserich @gooklani

Reply:
> We could nest this mixin a few levels deeper when applying to increase specificity and remove the "important" keyword as a result. What do you guys think? @kuserich @gooklani

I agree. However, this may not always be possible and we might need to be able to add an `!important` keyword. For now, I would suggest to remove it and in case there are enough use cases where `!important` is required, we could just extend the mixin with the `$important` flag. What do you think?

Reply:
Sure, I have removed the `important` keyword as per your suggestion.